samples/gibbs_ensemble/gibbs_ensemble.py

```diff
@@ -80,7 +80,8 @@ def send_info(self):
 
     def move_particle(self, DX_MAX):
         """ Move random particle inside the box """
-        self.old_particle = np.random.choice(self.system.part)
+        self.random_particle_id = np.random.choice(self.system.part[:].id)
+        self.old_particle = self.system.part[self.random_particle_id]
```

Comment:
why do we need to store the `random_particle_id` as a member of the object if it is not used elsewhere?

Reply:
Looks like I wrote that out of habit, I'll force update it not to store it as a member.